### PR TITLE
Generate more useful data that can be used with addons-frontend.

### PR DESCRIPTION
**Pull request by EnTeQuAk**
This is needed for mozilla/addons-nginx#7 and #9080 to be more useful.
The data that `generate_ui_test_addons` generates can directly be used
in the new frontend homepage.

This PR also stops seamonkey and thunderbird add-ons to be generated
by default.

### CODE REVIEW

File: src/olympia/landfill/management/commands/generate_ui_test_addons.py [hunk 1]
```diff
@@ -83,5 +91,3 @@ def handle(self, *args, **kwargs):
             for addon in hero_addons:
                 serializer.create_named_addon_with_author(addon)
             serializer.create_installable_addon()
-        cache.clear()
-        call_command('clear_cache')
```
Comment (EnTeQuAk) [inline src/olympia/landfill/management/commands/generate_ui_test_addons.py]: This is really old code that's pre cache-machine removal and still from when this was behind an API, which it isn't anymore.

The whole thing could be simplified and not written as a serializer but that's another cleanup.

File: Makefile-docker [hunk 2]
```diff
@@ -82,10 +82,13 @@ populate_data:
...
 	python manage.py reindex --wipe --force --noinput
 	python manage.py generate_addons --app firefox $(NUM_ADDONS)
-	python manage.py generate_addons --app thunderbird $(NUM_ADDONS)
 	python manage.py generate_addons --app android $(NUM_ADDONS)
-	python manage.py generate_addons --app seamonkey $(NUM_ADDONS)
 	python manage.py generate_themes $(NUM_THEMES)
+	# These add-ons are specifically useful for the addons-frontend
+	# homepage. You may have to re-run this, in case the data there
+	# changes.
+	python manage.py generate_ui_test_addons
```
Comment (eviljeff) [inline Makefile-docker]: can we rename the command something that more appropriately describes that it's used to populate addons-frontend locally, not just for the ui tests?

**Comment by jrbenny35** [comment]
Hey my stuff is useful elsewhere 😄

**Comment by jrbenny35** [comment]
But I do think I can expand it a bit more. I notice some categories that are featured on the homepage have no addons created for them.

**Pull request merged**